lib/SILOptimizer/Mandatory/OSLogOptimization.cpp

```diff
@@ -1402,7 +1403,7 @@ static SILInstruction *beginOfInterpolation(ApplyInst *oslogInit) {
   // formatting and privacy options are literals, all candidate instructions
   // must be in the same basic block. But, this code doesn't rely on that
   // assumption.
-  SmallPtrSet<SILBasicBlock *, 4> candidateBBs;
+  BasicBlockSetVector<4> candidateBBs(oslogInit->getFunction());
```

Comment:
Why is this specific one a BasicBlockSetVector instead of just a BasicBlockSet?

Reply:
Because `size()` and `begin()` is used. But I rewrite the code a bit so that BasicBlockSet can be used.

Reply:
Its ok. Can you at least add a comment to that effect with a fixme.